lib/src/extensions.dart

```diff
@@ -27,20 +27,16 @@ extension on int {
 extension on DateTime {
   /// Adds this DateTime and Duration and
   /// returns the sum as a new DateTime object.
-  DateTime operator +(Duration duration) {
-    return DateTime.fromMillisecondsSinceEpoch(millisecondsSinceEpoch + duration.inMilliseconds);
-  }
+  DateTime operator +(Duration duration) => DateTime.add(duration);
 
   /// Subtracts the Duration from this DateTime
   /// returns the difference as a new DateTime object.
-  DateTime operator -(Duration duration) {
-    return DateTime.fromMillisecondsSinceEpoch(millisecondsSinceEpoch - duration.inMilliseconds);
-  }
+  DateTime operator -(Duration duration) => DateTime.subtract(duration);
```

Comment:
Should this line be `DateTime operator -(Duration duration) => subtract(duration);`?